<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>764</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Comic Sans MS</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #252832;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="sidebar" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>261</width>
      <height>764</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #181D29;</string>
    </property>
    <widget class="QWidget" name="sidebar_content" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>55</y>
       <width>191</width>
       <height>461</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="sidebar_logo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/newPrefix/Downloads/1DAF91D6-CDA2-4EF4-8A2E-AA1482B02B86 3 1.png);
background-repeat: no-repeat;
background-size: cover;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="sidebar_name">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>12</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 24px; 
font-weight: 100;
color: #fff;</string>
      </property>
      <property name="text">
       <string>тимур</string>
      </property>
     </widget>
     <widget class="QLabel" name="sidebar_surname">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>116</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>99</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 24px;
font-weight: 800;
color: #fff;</string>
      </property>
      <property name="text">
       <string>КОВАЛЕВ</string>
      </property>
     </widget>
     <widget class="QWidget" name="nav" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>191</width>
        <height>271</height>
       </rect>
      </property>
      <widget class="QPushButton" name="all_films">
       <property name="geometry">
        <rect>
         <x>-20</x>
         <y>20</y>
         <width>171</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 24px; 
color: #fff;
border: none;</string>
       </property>
       <property name="text">
        <string>Все фильмы</string>
       </property>
      </widget>
      <widget class="QPushButton" name="settings">
       <property name="geometry">
        <rect>
         <x>-27</x>
         <y>60</y>
         <width>171</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 24px; 
color: #fff;
border: none;</string>
       </property>
       <property name="text">
        <string>Настройки</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scroll">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>-4</y>
      <width>1021</width>
      <height>775</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QScrollArea {
	border: none;
}</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scroll_content">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-349</y>
       <width>1021</width>
       <height>1124</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="main">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>1200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="block" native="true">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>310</y>
           <width>951</width>
           <height>750</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
            <layout class="QVBoxLayout" name="block_content_layout" />
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="header" native="true">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>35</y>
           <width>925</width>
           <height>191</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <widget class="QLabel" name="header_logo">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>351</width>
            <height>190</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(./images/logo.png);
background-repeat: no-repeat;
background-size: cover;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLineEdit" name="header_search">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>130</y>
            <width>770</width>
            <height>60</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #fff; 
border-radius: 18px;
padding: 0 20px;
padding-bottom: 5px;
font-size: 28px;
color: #000;</string>
          </property>
          <property name="placeholderText">
           <string>поиск...</string>
          </property>
         </widget>
         <widget class="QPushButton" name="header_button">
          <property name="geometry">
           <rect>
            <x>790</x>
            <y>130</y>
            <width>131</width>
            <height>60</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #224369;
color: #fff;
font-size: 28px;
border: none;
border-radius: 15px;
padding-bottom: 2px;</string>
          </property>
          <property name="text">
           <string>найти</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../userimage.qrc"/>
 </resources>
 <connections/>
</ui>
